packages/manager/modules/office/src/dashboard/translations/Messages_fr_FR.json

```diff
@@ -3,6 +3,8 @@
   "microsoft_office_license_office-proplus-pp": "Office Pro Plus",
   "microsoft_office_license_officeBusiness": "Office Business",
   "microsoft_office_license_officeProPlus": "Office Pro Plus",
+  "microsoft_office_license_microsoft-365-apps-for-business-pp-nce": "Microsoft Office 365 Apps for Business",
+  "microsoft_office_license_microsoft-365-apps-for-enterprise-pp-nce": "Microsoft Office 365 Apps for Enterprise",
```

Comment:
**nitpick:** Both aren't translations. Why not prefer using constant from here?

Reply:
The reason for not using it as a constant is, when there's a new plan 'ABC' we would see 'microsoft_office_license_ABC' in the dropdown till we add a new translation key in manager. If we maintain it as a constant, we would see an empty option till we add a constant for new plan.